src/m4st/callhome/process_mappings.py

```diff
@@ -0,0 +1,122 @@
+import os
+
+import pandas as pd
+from tqdm import tqdm
+
+
+def mapping_name_to_filename(mname: str):
+    return mname[3:] + ".cha"
+
+
+def process_line_ids(maybe_lines: str):
```

Comment:
There are quite a few functions without a specified return type, e.g.

```suggestion
def process_line_ids(maybe_lines: str) -> list[int]:
```

(I think)